cms/static/js/views/settings/main.js

```diff
@@ -436,6 +444,12 @@ define(['js/views/validation', 'codemirror', 'underscore', 'jquery', 'jquery.ui'
                        image_key = 'web_science_image_name';
                        image_path_key = 'web_science_image';
                        break;
+                   case 'upload-web-science-main-image':
+                       title = gettext('Upload your course main image.');
```

Comment:
Are you going to add appropriate translates or they have been added before?

Reply:
if we need a translation, we could use it in future, like another's gettext in this template.